helpers/properties_helper.rb

```diff
@@ -86,6 +86,13 @@ module PropertiesHelper
     # "246" => { :type => :resist, :name => "Piercing Resistance"}
   }
 
+  AETHERIA_SLOTS = {
+    0 => 0,
+    1 => 1,
+    3 => 2,
+    7 => 3
+  }
```

Comment:
Alternatively we could use binary literals to be more explicit about what the keys are.
```suggestion
  AETHERIA_SLOTS = {
    0b000 => 0,
    0b001 => 1,
    0b011 => 2,
    0b111 => 3
  }
```